tests/integration/Owncloud/OcisPhpSdk/OcisTest.php

```diff
@@ -127,6 +127,36 @@ public function testAddUserToGroup(): void
         }
     }
 
+    /**
+     * @return void
+     */
+    public function testRemoveExistingUserFromGroup(): void
+    {
+        $token = $this->getAccessToken('admin', 'admin');
+        $ocis = new Ocis($this->ocisUrl, $token, ['verify' => false]);
+        $users = $ocis->getUsers('admin');
+        $philosophyHatersGroup =  $ocis->createGroup("philosophy-haters", "philosophy haters group");
+        $this->createdGroups = [$philosophyHatersGroup];
+        $philosophyHatersGroup->addUser($users[0]);
```

Comment:
I think it would be good to have more than one user in the group, just to make sure the `removeUser` function not removing all users from the group, but the correct one only